pyuvdata/tests/test_utils.py

```diff
@@ -351,3 +351,55 @@ def test_deprecated_funcs():
                                                      uvversion.version],
                          category=DeprecationWarning,
                          message='The combine_histories function is deprecated')
+
+
+def test_redundancy_finder():
+    """
+        Confirm that get_baseline_redundancies returns baselines that have the same length.
+        Should probably check orientation too.
```

Comment:
Is this comment out of date? It looks like you are checking orientation because you're taking a dot product.

Reply:
Thanks, I missed that. I'll fix it.